app/controllers/item/additionalInformation/index/AdditionalInformationTypeController.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.item.additionalInformation.index
+
+import controllers.actions._
+import controllers.{NavigatorOps, SettableOps, SettableOpsRunner}
+import forms.SelectableFormProvider
+import models.{Index, LocalReferenceNumber, Mode}
+import navigation.{ItemNavigatorProvider, UserAnswersNavigator}
+import pages.item.additionalInformation.index.AdditionalInformationTypePage
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import repositories.SessionRepository
+import services.AdditionalInformationService
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import views.html.item.additionalInformation.index.AdditionalInformationTypeView
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class AdditionalInformationTypeController @Inject() (
+  override val messagesApi: MessagesApi,
+  implicit val sessionRepository: SessionRepository,
+  navigatorProvider: ItemNavigatorProvider,
```

Comment:
We will need a separate navigator for each additional information, but happy for this to be done in a future PR 🙂